climpred/bootstrap.py

```diff
@@ -340,12 +342,22 @@ def bootstrap_compute(
         results = results.drop(res_drop)
         ci = ci.drop(ci_drop)
     results = xr.concat([results, ci], 'results')
-    results['results'] = [
-        'skill',
-        'p',
-        'low_ci',
-        'high_ci',
-    ]  # add ci levels in metadata
+    results['results'] = ['skill', 'p', 'low_ci', 'high_ci']
+    metadata_dict = dict()
+    metadata_dict['confidence interval levels'] = str(ci_high) + ' ' + str(ci_low)
+    metadata_dict['bootstrap iterations'] = bootstrap
```

Comment:
I prefer no underscores replacing spaces

Reply:
@ahuang11 , what do you mean here?

Reply:
`bootstrap_iterations` rather than `bootstrap iterations` personal preference for anything related to coding

Reply:
usually people write plain english in netcdf attributes. therefore I want to leave this

Reply:
@aaronspring, all of the CMIP6 (CF-compliant) attributes use the underscore convention. I think it's best to keep it like that. From a computer science standpoint, it makes tab-completion and string matching easy since spaces are a nightmare on command line.

Reply:
Here's some CMIP6 attributes for instance

<img width="396" alt="Screen Shot 2019-08-05 at 9 49 25 AM" src="https://user-images.githubusercontent.com/8881170/62469465-54dfa180-b766-11e9-83b5-ef995f7bb9af.png">

Reply:
ups

Reply:
changed